yaml/argo/security/argo-workflow-parameter-command-injection.test.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
is the `apiVersion` key mandatory? You could add an additional `pattern-inside` to match `argoproj.io` to make sure the rule will report findings only for Argo yaml files.